src/Stormpath/Authc/UsernamePasswordRequest.php

```diff
@@ -22,12 +22,15 @@ class UsernamePasswordRequest implements AuthenticationRequest
     private $username;
     private $password;
     private $host;
+    private $accountStore;
 
-    public function __construct($username, $password, $host = null)
+    public function __construct($username, $password, array $options = array())
```

Comment:
You're right, I missed that point. I can add back the `$host` as parameter for the constructor and put the `$options` after it.